app/models/work.rb

```diff
@@ -0,0 +1,44 @@
+class Work < ApplicationRecord
+
+  has_many :votes
+  has_many :users, through: :votes
+
+  CATEGORIES = %w(album book movie) # For checkboxes on form partials // validations
+
+  validates :category,  presence: true,
+            inclusion: { in: CATEGORIES }
+
+  validates :title, presence: true,
+            uniqueness: { scope: :category }
+
+  def self.by_category(category)
+    if Work.all.any? { |work| work.category == category }
```

Comment:
This is a somewhat inefficient way to find this (you have to pull back all of the works just to check if a category exists) though without writing SQL it would be tricky to optimize.

(It's totally fine here but you will want to avoid patterns like this in production code during your internship.)